src/style.css

```diff
@@ -1,3 +1,86 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
 body {
+  font-family: Roboto, sans-serif;
+  margin: 0;
+  padding: 0;
+}
+
+.header {
+  width: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+  height: 60px;
+  flex-shrink: 0;
+  background: #FFF;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
   margin: 0;
+  padding: 0;
+  border: 0;
+  display: block;
+  }
+
+li {
+  display: inline;
+  height: 60px;
+  margin: 0 20px;
```

Comment:
avoid margin collapsing. add margin-right instead to each of item besides last one